extensions/manage/src/lib.rs

```diff
@@ -145,3 +145,9 @@ pub trait ManageClient: ExtensionClient<ManageExtension> {
 }
 
 impl<C: ExtensionClient<ManageExtension>> ManageClient for C {}
+
+/// Empty file written to mark that a file system should be reformatted
+///
+/// When booting, if this file is written to a filesystem, it means that the previous
+/// power cycle was caused by a factory reset and the storage should be reformatted
```

Comment:
nit: This sounds like the mechanism works for all storages, but AFAIS it is only implemented for the EFS.